charts/flyte-core/values-onprem.yaml

```diff
@@ -0,0 +1,841 @@
+# ---------------------------------------------------------------------
+# Core System settings
+# This section consists of Core components of Flyte and their deployment
+# settings. This includes FlyteAdmin service, Datacatalog, FlytePropeller and
+# Flyteconsole
+# ---------------------------------------------------------------------
+
+#
+# FLYTEADMIN SETTINGS
+#
+
+flyteadmin:
+  enabled: true
+  # -- Replicas count for Flyteadmin deployment
+  replicaCount: 1
+  image:
+    # -- Docker image for Flyteadmin deployment
+    repository: cr.flyte.org/flyteorg/flyteadmin # FLYTEADMIN_IMAGE
+    tag: v1.1.29-hotfix # FLYTEADMIN_TAG
+    pullPolicy: IfNotPresent
+  # -- Additional flyteadmin container environment variables
+  #
+  # e.g. SendGrid's API key
+  #  - name: SENDGRID_API_KEY
+  #    value: "<your sendgrid api key>"
+  #
+  # e.g. secret environment variable (you can combine it with .additionalVolumes):
+  # - name: SENDGRID_API_KEY
+  #   valueFrom:
+  #     secretKeyRef:
+  #       name: sendgrid-secret
+  #       key: api_key
+  env: []
+  # -- Default resources requests and limits for Flyteadmin deployment
+  resources:
+    limits:
+      cpu: 250m
+      ephemeral-storage: 100Mi
+      memory: 500Mi
+    requests:
+      cpu: 10m
+      ephemeral-storage: 50Mi
+      memory: 50Mi
+  # -- Default regex string for searching configuration files
+  configPath: /etc/flyte/config/*.yaml
+  # -- Initial projects to create
+  initialProjects:
+    - flytesnacks
+    - flytetester
+    - flyteexamples
+  # -- Service settings for Flyteadmin
+  service:
+    annotations:
+      projectcontour.io/upstream-protocol.h2c: grpc
+    type: ClusterIP
+    loadBalancerSourceRanges: []
+  # -- Configuration for service accounts for FlyteAdmin
+  serviceAccount:
+    # -- Should a service account be created for flyteadmin
+    create: true
+    # -- Should a service account always be created for flyteadmin even without an actual flyteadmin deployment running (e.g. for multi-cluster setups)
+    alwaysCreate: false
+    # -- Annotations for ServiceAccount attached to Flyteadmin pods
+    annotations: {}
+    # -- ImagePullSecrets to automatically assign to the service account
+    imagePullSecrets: []
+    # -- Should a ClusterRole be created for Flyteadmin
+    createClusterRole: true
+  # -- Annotations for Flyteadmin pods
+  podAnnotations: {}
+  # -- nodeSelector for Flyteadmin deployment
+  nodeSelector: {}
+  # -- tolerations for Flyteadmin deployment
+  tolerations: []
+  # -- affinity for Flyteadmin deployment
+  affinity: {}
+  secrets: {}
+  additionalVolumes: 
+    - configMap:
+        name: sslcerts
+      name: sslcerts
+  additionalVolumeMounts: 
+    - mountPath: /etc/ssl/certs/
+      name: sslcerts
```

Comment:
Can you add a few comments regarding this config and how is this different than the L78-L81

Reply:
Ok  I will do